CHANGES_NEXT_RELEASE

```diff
@@ -1,3 +1,4 @@
+- Fix: Log trace signals: notifications payload request/response #2895
```

Comment:
```suggestion
- Fix: Log trace signals: notifications payload request/response (#2895)
```

Reply:
> Fix: Log trace signals: notifications payload request/response (#2895)

Fixed in https://github.com/telefonicaid/fiware-orion/pull/3854/commits/6d10fe4222c95df14ed087756b6188681ad188f1